### PR TITLE
Initial attempt to get rid of memory leaks with and without navigator

### PR DESCRIPTION
**Pull request by JoFrMueller**
@iangilman I managed to get rid of all OSD object instances. Could you have a glance at the PR and tell me what of it is okay with you and what we should adjust further? Thanks!

### CODE REVIEW

File: src/navigator.js [hunk 1]
```diff
@@ -406,6 +406,15 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
         return $.Viewer.prototype.addTiledImage.apply(this, [optionsClone]);
     },
 
+    destroy: function() {
+        this.element.getElementsByTagName('div')[0].removeChild(this.displayRegionContainer);
+        this.displayRegionContainer.removeChild(this.displayRegion);
+        this.viewer.removeControl(this.element);
+        this.innerTracker.destroy();
+        this.outerTracker.destroy();
+        this.element = null;
```
Comment (iangilman) [inline src/navigator.js:415]: Presumably we need this too:

```
return $.Viewer.prototype.destroy.apply(this);
```

Once that's in, we probably don't need to clean up the trackers or the element ourselves either.

File: LEAKS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Memory leaks
```
Comment (iangilman) [inline LEAKS.md:1]: This is good info, but I don't think it belongs in the project. Please remove before we merge.

**Comment by JoFrMueller** [comment]
I'm quite busy now but will try to provide a PR for #1721 towards the weekend.

**Comment by JoFrMueller** [comment]
Cleanup provided separated by #1832.

**Pull request closed without merging**